sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -46,8 +46,9 @@ Once you have the values for endpoint and subscription key, you can create the [
 
 ```C# Snippet:CreateTextAnalyticsClient
 string endpoint = "<endpoint>";
-string subscriptionKey = "<subscriptionKey>";
-var client = new TextAnalyticsClient(new Uri(endpoint), subscriptionKey);
+TextAnalyticsSubscriptionKeyCredential credential = "<credential>";
+var credential = new TextAnalyticsSubscriptionKeyCredential(subscriptionKey);
```

Comment:
Depending on where we land with the constructors -- did we decide to keep the string constructors? -- let's either put the string constructor here with a comment that you can roll a credential with another constructor, or put snippets for both and call out the difference.

Reply:
For now I'm going with 4 constructors. 
After UX study, if wanted, we can add the string constructor